proto/vectorindex.proto

```diff
@@ -112,25 +189,25 @@ message _SearchAndFetchVectorsResponse {
 }
 
 message _GetItemBatchRequest {
-  string index_name = 1;
-  repeated string ids = 2;
-  _MetadataRequest metadata_fields = 3;
+    string index_name = 1;
+    repeated string ids = 2;
+    _MetadataRequest metadata_fields = 3;
```

Comment:
Did you run a code formatter or was this intended?

Reply:
yes, if you check the above indentation, only these are wrong.